src/dpw_types.jl

```diff
@@ -142,13 +142,13 @@ function DPWSolver(;depth::Int=10,
                     check_repeat_action::Bool=true,
                     tree_in_info::Bool=false,
                     rng::AbstractRNG=Random.GLOBAL_RNG,
-                    estimate_value::Any = RolloutEstimator(RandomSolver(rng)),
-                    init_Q::Any = 0.0,
-                    init_N::Any = 0,
-                    next_action::Any = RandomActionGenerator(rng),
-                    default_action::Any = ExceptionRethrow(),
-                    reset_callback::Function = (mdp, s)->false,
-                    show_progress::Bool = false,
+                    estimate_value::Any=RolloutEstimator(RandomSolver(rng)),
```

Comment:
In the future, please try to avoid small formatting changes unrelated to the PR, it makes it harder to review.

Reply:
Will do, thanks!